app/boards/shields/elephant42/elephant42.keymap

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+ 
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/outputs.h>
+
+#define LOWR 1
+#define RAIS 2
+#define ADJT 3
+
+/ {
+	keymap {
+		compatible = "zmk,keymap";
+
+		default_layer {
```

Comment:
The formatting seems a bit crazy in GitHub. Maybe an issue with tabs size in your editor vs. GitHub?

Reply:
This was caused by me trying to use the Zephyr VS code plugin plus the C lang formatter. In the end, I had to disable both since they did more harm than good.

Reply:
Hey @filoxo it seems this file has gotten even bigger spaces between the keys. Could you take another look at it? I would recommend using spaces to align keymaps

Reply:
Okay, finally got around to this. I think I actually got the formatting right this time.